build.rs

```diff
@@ -25,10 +25,13 @@ const MIN_VERSION: &'static str = "1.37.0-nightly";
 
 lazy_static! {
     static ref PYTHON_INTERPRETER: &'static str = {
-        ["python3", "python"]
+        ["python", "python3"]
             .iter()
-            .map(|bin| (bin, Command::new(bin).spawn()))
-            .find(|(_, r)| r.is_ok())
+            .map(|bin| (bin, Command::new(bin).arg("--version").output()))
```

Comment:
How about this:
```rust
            .find(|bin| {
                if let Ok(out) = Command::new(bin).arg("--version").output() {
                    out.stdout.starts_with(b"Python 3")
                } else {
                    false
                }
            })
            .expect("Python 3.x interpreter not found")
```

Reply:
Thank you for comment!
Travis show me one curious things, Windows Anaconda3 Python output version by stderr. Python provided by official output version by stdout, only Anaconda3.
So It looks Anaconda3 bug. No one noticed.

Reply:
Ah, OK, thank you for investigating that.
My intention is just to shorten codes, though.

Reply:
No problem, I feel method chain is not readable code.
I changed to something like your suggestion.